test/let.txt

```diff
@@ -0,0 +1,42 @@
+;; source code for let.wasm, which was created by wasp
```

Comment:
Is this file just a drive-by fix because it was previously missing?

Reply:
Yes. (It was useful to debug this.)